lib/msal-browser/src/custom_auth/sign_in/auth_flow/SignInScenario.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export enum SignInScenario {
```

Comment:
Please use constants instead of enums for bundle minitifaction. Example: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/76be68e5e3d042dff49fdc3b9ba6def60aea7f8e/lib/msal-common/src/utils/Constants.ts#L96